### PR TITLE
[Feature] Adding loan record chart

### PR DESCRIPTION
**Pull request by Jeevan1**

### CODE REVIEW

File: src/pages/UserProfile.jsx [hunk 1]
```diff
@@ -1,129 +1,264 @@
...
+  }
 `;
 
+const UserRecordSummarizedWrapper = styled.div`
+  margin: 20px;
+  width: fit-content;
+  padding: ${({ theme }) => theme.spacing.s16};
+  border: 1px solid ${({ theme }) => theme.palette.border.secondary};
+  border-radius: ${({ theme }) => theme.borderRadius.container};
+  background-color: white;
```
Comment (RahulRimal) [inline src/pages/UserProfile.jsx:58]: use get color from theme

File: src/pages/UserProfile.jsx [hunk 2]
```diff
@@ -1,129 +1,264 @@
...
+`;
 
+const UserRecordSummarizedHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 20px;
```
Comment (RahulRimal) [inline src/pages/UserProfile.jsx:65]: use theme here

File: src/pages/UserProfile.jsx [hunk 3]
```diff
@@ -1,129 +1,264 @@
...
+  "phone_number",
+  "dob",
+];
 
+const UserRecordSummarized = () => {
+  const theme = useTheme();
 
+  const option = {
+    grid: { top: 20, right: 40, bottom: 50, left: 40 },
+    // textStyle: { color: "red" },
```
Comment (RahulRimal) [inline src/pages/UserProfile.jsx:85]: remove commented code

**Review by RahulRimal — CHANGES_REQUESTED** [review]

File: src/components/UserIconButton.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+function UserIconButton({
+  type,
+  onClick,
+  onHover,
+  showBgOnHover = true,
+  children,
+}) {
+  return (
+    <IconButtonWrapper
```
[the remark this answers is not in the document]
Comment (Jeevan1) [reply src/components/UserIconButton.jsx:24]: done

File: src/pages/UserProfile.jsx (hunk 1 above)
Comment (Jeevan1) [reply src/pages/UserProfile.jsx]: done

File: src/pages/UserProfile.jsx (hunk 2 above)
Comment (Jeevan1) [reply src/pages/UserProfile.jsx]: done

File: src/pages/UserProfile.jsx (hunk 3 above)
Comment (Jeevan1) [reply src/pages/UserProfile.jsx]: done

**Review by RahulRimal — APPROVED** [review]

**Pull request merged**